azure-stack/hci/manage/manage-secure-baseline.md

```diff
@@ -0,0 +1,117 @@
+| Data at-rest encryption    | [BitLocker for OS boot volume](/windows/security/information-protection/bitlocker/bitlocker-overview)          | Encrypts the OS startup volume on each server.                                        | No                              |
+| Data at-rest encryption    | [BitLocker for data volumes](/windows/security/information-protection/bitlocker/bitlocker-overview)            | Encrypts cluster shared volumes (CSVs) on this cluster                               | No                              |
+| Data in-transit protection | [Signing for external SMB traffic](/troubleshoot/windows-server/networking/overview-server-message-block-signing)      | Signs SMB traffic between this system and others to help prevent relay attacks.       | Yes                             |
+| Data in-transit protection | [SMB Encryption for in-cluster traffic](/windows-server/storage/file-server/smb-security#smb-encryption) | Encrypts traffic between servers in the cluster (on your storage network).            | No                              |
+
+### Modify security after deployment
+
+Once the deployment is complete, you can modify security features while maintaining drift control. Some features require a reboot to take effect.
+
+### PowerShell cmdlet properties
+
+The following cmdlet properties are for the *AzureStackOSConfigAgent* module.
+
+- `Get-AzsSecurity`  -Scope: <Local | PerNode | AllNodes | Cluster>
+- `Enable-AzsSecurity`   -Scope <Local | Cluster>
+- `Disable-AzsSecurity`  -Scope <Local | Cluster>
+
+  - **Local** - Provides boolean value (true/False) on local node. Can be run from a regular remote PowerShell session.
+  - **PerNode** - Provides boolean value (true/False) per node.
+  - **Report** - Requires CredSSP or an Azure Stack HCI server using a remote desktop protocol (RDP) connection.
+    - AllNodes –Provides boolean value (true/False) computed across nodes-  requires CredSSP (when using remote PowerShell) or Console session (RDP).
+    - Cluster  –Provides boolean value from ECE store. Interacts with the orchestrator and acts to all the nodes in the cluster, requires deployment authorization (PREFIX-ECESG) and either CredSSP (when using remote PowerShell) or Console session (RDP).
+  - **FeatureName** - <CredentialGuard | DriftControl | DRTM | HVCI | SideChannelMitigation | SMBEncryption | SMBSigning | VBS>
+    - Credential Guard
+    - Drift Control
+    - VBS (Virtualization Based Security)- We only support enable command.
+    - DRTM (Dynamic Root of Trust for Measurement)
+    - HVCI (Hypervidor Enforced if Code Integrity)
+    - Side Channel Mitigation
+    - SMB Encryption
+    - SMB Signing
+
```

Comment:
Can the cmdlet properties be formatted as a table?

Reply:
The table below should have an intro sentence.

Reply:
@alkohli - Working on a table, but sample commands and output would be most helpful.